tests/data/config.yaml

```diff
@@ -26,4 +26,6 @@ usage_metrics:
       min: 1001
     min_consumption: 5
     usage_aggregation: average
-version: 1.1.1
\ No newline at end of file
+logging:
+  level: WARN
+version: 1.1.1
```

Comment:
Unrelated to this PR (feel free to ignore) but this is an old incorrect version format. `1` would be a valid format.

Reply:
I kept the same version in the testing data,
I understand the importance of the version 
for now is that it is present in the file ?

I could add a PR to check the format is valid
if that's what you mean

Reply:
No I just noticed it was wrong. Fine to leave it as is for now.